src/main/java/org/apache/cassandra/metrics/TableMetrics.java

```diff
@@ -400,6 +401,18 @@ public boolean isPropertyListPattern() {
         public boolean isPropertyValuePattern() {
             return false;
         }
+        
+        /**
+         * This type is not really serializable. 
+         * Replace it with vanilla objectname.
+         */
+		private Object writeReplace() throws ObjectStreamException {
+			try {
+				return new ObjectName(getDomain(), getKeyPropertyList());
+			} catch (MalformedObjectNameException e) {
+				throw new InvalidObjectException(toString());
+			}
+		}
```

Comment:
Did you discover the one true indentation style -- tabs?

Reply:
Meh. Ran a new eclipse. We should check the formatter in with the maven project.